jenkins/opensearch-dashboards/distribution-build.jenkinsfile

```diff
@@ -1008,6 +1014,22 @@ def triggerBWCTests(String buildManifestUrl, String platform, String distributio
     }
 }
 
+def triggerDistributionValidationWorkflow(String version) {
+    def triggerValidationWorkflow =
+            build job: 'distribution-validation',
+            propagate: true,
+            wait: true,
+            parameters: [
+                string(name: 'VERSION', value: version),
+                string(name: 'OS_BUILD_NUMBER', value: 'latest'),
+                string(name: 'OSD_BUILD_NUMBER', value: BUILD_NUMBER),
+                string(name: 'OPTIONAL_ARGS', value: 'using-staging-artifact-only'),
+                string(name: 'DOCKER_SOURCE', value: 'dockerhub'),
```

Comment:
We can ignore this parameter Sayali, DOCKER_SOURCE defaults to dockerhub https://github.com/opensearch-project/opensearch-build/blob/c33d51b133c909927d62d9e0cebdba320b19fd71/src/validation_workflow/validation_args.py#L80

Reply:
Its a choice in jenkins workflow. And defaults whatever is first choice. In case we change the order in future, it would default to whatever is first. Hence added this